docs/sources/visualizations/traces.md

```diff
@@ -0,0 +1,19 @@
++++
```

Comment:
It might be worth taking some of this content into a `README.md` for the plugin, so that the plugin details view has something descriptive for the user. WDYT?